provider-tutorials/provider-cli.md

```diff
@@ -214,13 +214,150 @@ For `--network`you have two options, either `mainnet` or `rinkeby`. For `--to-ad
 
 `ya-provider` allows to fine-tune the config created with `golemsp settings` using commands `config`, `preset`, `profile`, and `exe-unit`.
 
-Additionally, it enables configuration of the certificate keystore and of the domain whitelist, which are used to control what kind of outbound traffic is allowed out of the VM containers run by the requestors on your machine.
+Additionally, it enables configuration of the rules, certificate keystore and domain whitelist, which are used to control what kind of outbound traffic is allowed out of the VM containers run by the requestors on your machine.
+
+### Rules
+
+`ya-provider` offers a mechanism that allows to control the Rules for specific features i.e. Outbound Network.
+
+#### Outbound Network feature
+
+This feature defines rules and modes for Outbound Network.
+It gives provider control over which entities are trusted and what level of Outbound Network access they have.
+
+Outbound feature provides three rules:
+
+1. `everyone`: defines the default outbound access for all requestors.
+2. `audited-payload`: controls outbound access for manifests which are audited and signed by a trusted party.
+   Trust is gained when provider sets this rule for root certificate of auditing organisation.
+   Provider can have multiple `audited-payload` rules for different `X.509` certificates.
+   To match this rule, requestor should get his manifest audited and signed by organisation which is trusted by provider.
+   It doesn't matter from which requestor node request is made.
+3. `partner`: allows outbound access for any manifests, as long as requestor node is a partner to trusted organisation.
+   Trust is gained when provider sets Partner rule for root certificate of trusted organisation.
```

Comment:
```suggestion
   Trust is gained when provider sets Partner rule for root `Golem` certificate of trusted organisation.
```

Reply:
There is nowhere stated that Partner rule supports only `Golem`, and Audited Payload only `X.509`, so adding here "`Golem` certificate" will at least discourage setting Partner rule to `X.509` certificate.